eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -23,6 +25,11 @@ the old OS deployment is still running.
 .SH OPTIONS
 .IX Header "OPTIONS"
 .\"
+.IP "\fB\-\-dry-run\fP"
+Print to stdout the actions that have been applied and the actions left to
```

Comment:
No need to mention ‘stdout’. How about “Print the actions that have been applied…”?

Reply:
If you just run it on the command line the output is pretty messy because there's other stuff on stderr

Reply:
OK.